mitogen/parent.py

```diff
@@ -479,7 +482,10 @@ def openpty():
 
     master_fp = os.fdopen(master_fd, 'r+b', 0)
     slave_fp = os.fdopen(slave_fd, 'r+b', 0)
-    disable_echo(master_fd)
+    try:
+      disable_echo(master_fd)
+    except:
```

Comment:
I won't accept a bare `except:` or `except Exception:`.

1. What specific failure(s) are you trying to catch?
2. Is ignoring the exception an acceptable fallback in the general case?

Reply:
I can make that Solaris-specific. The failure is definitely a documented behavior on the solaris platforms.